fortran-inference/inference_scaling_imagenet.F90

```diff
@@ -43,36 +44,43 @@ program main
 use_cluster = get_env_var("SS_CLUSTER", .false.)
```

Comment:
I had added a comment in the past that said "Is "SR_LOG_FILE": "srlog.out" and "SR_LOG_LEVEL": "info" needed here?" under "!Retrieve all environment variables" - add if you think this is still applicable!

Reply:
These are not currently used/set via the Fortran client constructor. In the future, if we decide to use ConfigOptions object, then we could certainly add these then.